template/6-non_functional_requirements.md

```diff
@@ -2,22 +2,46 @@
 
 ## Security, privacy, and data retention policies
 
-*Which are the applicable laws and regulations?*
-- Regulation from [Google Map API geolocation](https://developers.google.com/maps/documentation/geolocation/policies?hl=en)
```

Comment:
Do you really use this in what you added?

Reply:
You're right, I had added that for the outline.
But writting the non functional requirements, I didn't felt the need to use it.

Reply:
Ok, this makes sense!